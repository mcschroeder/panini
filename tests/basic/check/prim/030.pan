f30 : {v:unit|v=unit}
f30 = unit

f31 : {v:bool|v=true}
f31 = true

f32 : {v:int|v=1}
f32 = 1

f33 : {v:string|v="hello"}
f33 = "hello"

f34 : {v:char|v='a'}
f34 = 'a'

f35 : {v:string|v="a"}
f35 = "a"
