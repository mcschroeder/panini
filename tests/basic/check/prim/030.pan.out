f30 : {v:𝟙 | v = unit}
f31 : {v:𝔹 | v = true}
f32 : {v:ℤ | v = 1}
f33 : {v:𝕊 | v = "hello"}
f34 : {v:ℂ𝕙 | v = 'a'}
f35 : {v:𝕊 | v = "a"}